examples/dreamcast/vmu/vmu_beep/beep.c

```diff
@@ -0,0 +1,195 @@
+/*  VMU Beep for KallistiOS ##version##
+
+    beep.c
+    (C) 2004 SinisterTengu
+    (C) 2008 Donald Haase
+
+*/
+
+/*
+    This example allows you to send raw commands to the VMU's buzzer, using the CLOCK
+    function implemented by the VMU.
+
+    On a typical VMU, which only has a single channel, only the low 2 bytes are used to
+    create a single waveform, with the lowest byte being the period of the waveform, and
+    the next byte being the duty cycle. Duty cyle should always be less than the period
+    and is recommended to stay around 50%.
+
+    All of the interface code for this was stolen from SinisterTengu's Puru Puru Demo from '04
+    (when Kamjin first RE'd the puru). The rest was based off the puru driver as well as other bits
+    of maple driver and lots of guesswork (and messed up vmu).
+
+    Try out this value: 0x000065F0
+
+    Enjoy, and let me know about some interesting sounds.
+ */
+
+#include <dc/maple.h>
+#include <dc/maple/controller.h>
+#include <dc/maple/vmu.h>
+#include <plx/font.h>
+
+#include <stdio.h>
+#include <stdint.h>
+
+extern uint8_t romdisk[];
+KOS_INIT_FLAGS(INIT_DEFAULT);
+KOS_INIT_ROMDISK(romdisk);
+
+int main(int argc, char *argv[]) {
+    maple_device_t *dev, *vmudev;
+    cont_state_t *state;
+    plx_font_t *fnt;
+    plx_fcxt_t *cxt;
+    point_t w;
+    int i = 0, count = 0;
+    uint16_t old_buttons = 0, rel_buttons = 0;
+    uint32_t effect = 0;
+    uint8_t n[8] = { 0, 0, 0, 0, 0, 0, 0, 0 }; //nibbles
+    char s[8][2] = { "", "", "", "", "", "", "", "" };
+
+    /*if these buttons are pressed, exit_app */
+    cont_btn_callback(0, CONT_START | CONT_A | CONT_B | CONT_X | CONT_Y,
+        (void(*)(uint8_t, uint32_t))arch_exit);
+
+    pvr_init_defaults();
+
+    fnt = plx_font_load("/rd/axaxax.txf");
+    cxt = plx_fcxt_create(fnt, PVR_LIST_TR_POLY);
+
+    pvr_set_bg_color(0.0f, 0.0f, 0.0f);
+
+    for (;;) {
```

Comment:
I meant remove the spaces, not add them.
```suggestion
    for(;;) {
```